template/ios/RNBaseProjectTypeScript.xcodeproj/project.pbxproj

```diff
@@ -592,6 +593,7 @@
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
+				CXX = "";
```

Comment:
why we need add CXX, LD, LDPLUSPLUS = ""?

Reply:
because the react native upgrader helper recommend that, this file only follows it